package.json main is pointing to null

in package.json main field is pointing to "main": "dist/index.js",   but dist folder is not shipped with NPM in this plugin. 

using plugin fails with following exception:
```
Error: Cannot find module 'rollup-plugin-filesize'
```

usage:
```javascript
import filesize from 'rollup-plugin-filesize';
```

ps. It was working fine few days ago

--- a/package.json
+++ b/package.json
@@ -15,8 +15,8 @@
     "lint": "eslint .",
     "test": "nyc ava test/index.test.js",
     "remove-dist": "rimraf dist/*",
-    "pretest": "npm run remove-dist && rollup -c",
-    "prepare": "npm run test"
+    "pretest": "rollup -c",
+    "prepare": "npm run remove-dist && npm run test"
   },
   "engines": {
     "node": ">=10.0.0"

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "prepare": "npm run test"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
   },
   "repository": {
     "type": "git",

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -24,7 +24,7 @@ export default [
 			babel({
 				babelrc: false,
 				plugins: ["@babel/plugin-syntax-import-meta"],
-				presets: [["@babel/preset-env", { targets: { node: 8 } }]],
+				presets: [["@babel/preset-env", { targets: { node: 10 } }]],
 			}),
 			filesize({
 				showBeforeSizes: "release",
@@ -42,7 +42,7 @@ export default [
 			plugins: [
 				babel({
 					babelrc: false,
-					presets: [["@babel/preset-env", { targets: { node: 8 } }]],
+					presets: [["@babel/preset-env", { targets: { node: 10 } }]],
 				}),
 				filesize({
 					showBeforeSizes: "release",

